### ISSUE TITLE
Use the `fetch` package only

### ISSUE THREAD

**Issue by julien51** [unlock-protocol/unlock#8853]
In our codebase, we currently use multiple HTTP client packages: `fetch`, `node-fetch`, `cross-fetch` and `axios`...
This is adding unnecessary complexity (cognitive load as these API differ slightly), extra build time, heavier bundles... etc.

We should normalize to _only_ use `fetch` is nos supported by all major browsers and node.js natively (as of 18... which we don't support yet but would soon).

**Comment by jordang7** [comment]
I will look into working on this! 

Curious is there a set amount for the bounty? I can only see the bounty tag on the issue but no other details

**Comment by julien51** [comment]
@jordang7 Thanks for the follow up! Let's say 500 DAI!

**Comment by julien51** [comment]
@jordang7 any progress?

**Comment by jordang7** [comment]
Yes, sorry was a little more busy than expected and had some trouble getting setup. I've replaced all HTTP client packages with fetch currently. Having some trouble transitioning the tests to mock fetch instead of axios, specifically the wedlockService test. Having 4 tests fail there because of differing return structures. Haven't found a fix  for this yet

![wedlocks-error](https://user-images.githubusercontent.com/32372824/175119304-e825efa8-222b-431f-8cb5-f91af800c619.png)
